zipline-gradle-plugin/src/test/projects/multipleJsTargets/build.gradle.kts

```diff
@@ -8,11 +8,10 @@ buildscript {
   }
   dependencies {
     classpath("app.cash.zipline:zipline-gradle-plugin:${project.property("ziplineVersion")}")
-    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10")
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.0")
```

Comment:
https://github.com/cashapp/zipline/issues/888

Reply:
Ah, I didn't mean you had to do it now!

Reply:
Oh sorry! Well it's resolved now haha